README.Rmd

```diff
@@ -41,25 +41,25 @@ library("googledrive")
 ### Package idiosyncrasies
 
 * All functions begin with the prefix `drive_`
-* Functions and parameters attempt to mimic local file navigating conventions in `R`, such as `list.files`.
+* Functions and parameters attempt to mimic local file navigating conventions in R, such as `list.files`.
 
 ### Quick demo
 
 Here's how to list the most recently modified 100 files on your drive. This will kick off your authentication, so you will be sent to your browser to authorize your Google Drive access. The functions here are designed to be pipeable, using `%>%`, however they can also be implemented without.
 
-```{r, eval = FALSE}
+```{r}
 drive_list()
 ```
 
-You can narrow the query by specifying a `path` and/or `pattern` you'd like to search within. For example, to search within the folder `foo` for a file named `bar` you could run the following.
+You can narrow the query by specifying a `path` and/or `pattern` you'd like to search within. For example, to search within the folder `foobar` for a file named `baz` you could run the following.
```

Comment:
This makes me wonder, as a user, why can't I just provide `foobar/baz`? What would be the difference? 🤔 I suspect answering this has implications for `drive_list()` docs and/or logic. Maybe put this in the existing issue thread for `drive_list()` interface? It has nothing to do with the README.